apps/src/maze/StepButton.jsx

```diff
@@ -0,0 +1,20 @@
+var msg = require('./locale');
+
+var StepButton = function (props) {
+  var classes = 'launch float-right';
+  if (!props.showStepButton) {
+    classes += ' hide';
+  }
+  return (
+    <button id="stepButton" className={classes}>
```

Comment:
This also looks like it should be protected at some level, until it owns its own click handler binding.